.github/ISSUE_TEMPLATE/bug-report---.md

```diff
@@ -0,0 +1,45 @@
+---
+name: "Bug report \U0001F41B"
```

Comment:
Also... i don't know why that `\U0001F41B` is in there? Maybe this was auto generated and it is that ----? Would prob remove unless there's good reason for it.

Reply:
Thats the 🐛  emoji.  Maybe for bringing levity and joy to bug reporter?  😄

Reply:
ooooh. i'm down with that.